test/functional/tools/metadata_column_names_conditional.xml

```diff
@@ -0,0 +1,30 @@
+<tool id="metadata_columns_conditional" name="metadata_columns_conditional" version="1.0.0">
+    <description>Tests whether metadata is being set correctly.</description>
+    <command><![CDATA[
+        cp '$input' '$output'
+    ]]></command>
+    <inputs>
+        <param name="input" type="data" multiple="false" />
+	<param name="variable" type="integer" value="1" size="1" />
+    </inputs>
+    <outputs>
+        <data format="tabular" name="output">
+            <actions>
+       	        <conditional name="variable">
+		    <when value="1">
```

Comment:
@pvanheus can you fix the indentation here and below?

Reply:
I've committed a version with new indentation - is this better?